src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -231,6 +231,11 @@ module GeneratedAttributes =
     let LoadedViaTestNameInsteadOf = "__LoadedViaTestNameInsteadOf__"
 
 
+/// contains reserved names for command line arguments
+module CommandLineArguments = 
+    let SimulatorOption = "simulator"
```

Comment:
done

Reply:
The short name is not accessible separately from the list of all short names. The way that the option is created in the standalone executable is:

```
new[] { "--simulator", "-s" }
```

It would be good to have one property for the long name and one for the short name in `CommandLineArguments`. In general, the short name is not always the first letter of the long name, so I think it would be better not to programmatically create the short names.